docs/farming-&-staking/farming/advanced-cli/cli-install.mdx

```diff
@@ -75,8 +75,8 @@ Your insights on our releases elevate our progress. Fill out this [form](https:/
 :::note Learn More About Syncing, Plotting & Farming
 
 If you would like to learn more about how the syncing, plotting and farming processes works on the Subspace Network, please refer to our Subnomicon links below.
-- [Synchronization](https://subnomicon.subspace.network/docs/network/network_protocols/#synchronization)
-- [Plotting](https://subnomicon.subspace.network/docs/consensus/consensus/plotting)
-- [Farming](https://subnomicon.subspace.network/docs/consensus/consensus/farming)
+[Synchronization](https://subnomicon.subspace.network/docs/network/network_protocols/#synchronization)
```

Comment:
One last thing - need to add <br / > at the end of each line, otherwise it's all in one line.